backend/migrations/2023-05-29-163141_remove_map_versions/down.sql

```diff
@@ -0,0 +1,7 @@
+-- This file should undo anything in `up.sql`
+CREATE TABLE map_versions (
```

Comment:
As stated in the comment in the first row, a `down.sql` file should undo every change made in the `up.sql` file. Since the `map_versions` table was removed in `up.sql`, the `down.sql` file logically had to include the statement to create the table.